stdpopsim/slim_engine.py

```diff
@@ -1189,6 +1340,7 @@ def simulate(
         slim_scaling_factor=1.0,
         slim_burn_in=10.0,
         dry_run=False,
+        recap_and_rescale=True,
```

Comment:
I see this is useful in testing, but how about we make it an undocumented option (`_recap_and_rescale`) so we don't have to maintain it?

Reply:
done